erpnext/accounts/page/pos/pos.js

```diff
@@ -1757,9 +1757,13 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 	mandatory_batch_no: function () {
 		var me = this;
 		if (this.items[0].has_batch_no && !this.item_batch_no[this.items[0].item_code]) {
-			frappe.throw(__(repl("Error: Batch no is mandatory for item %(item)s", {
-				'item': this.items[0].item_code
-			})))
+			frappe.prompt([
+             {'fieldname': 'batch', 'fieldtype': 'Select', 'label': __('Batch No'), 'reqd': 1, 'options':this.batch_no_data[this.items[0].item_code]}
```

Comment:
Hi @rmehta ,

Sorry about these, it only shows up when pushing to Github.

I am using Atom as an editor.

I will try to avoid it in the future...